assets/sass/components/adminbar/_googlesitekit-adminbar.scss

```diff
@@ -116,4 +116,9 @@
 			pointer-events: auto;
 		}
 	}
+
+	// Hides Widgets in No Data state
+	.googlesitekit-plugin .googlesitekit-adminbar .mdc-layout-grid__inner .googlesitekit-hidden {
+		display: none;
+	}
```

Comment:
This shouldn't be necessary to add here. If this is needed for specificity, I think we can simply add `!important` to `googlesitekit-hidden` since that should never be overridden.

Reply:
Removed and updated the base class.